packages/bot-web-ui/src/pages/main/main.scss

```diff
@@ -150,16 +121,14 @@
                 height: 4.8rem;
                 padding: 0 2.4rem;
                 width: auto;
-                display: flex;
-                justify-content: center;
-                align-items: center;
+                @include flex-center;
 
                 &__loaded {
+                    text-align: left;
+                    padding-left: 1.5rem;
```

Comment:
as per BEM `__loaded` is not an element, it is a modifier and should be `--loaded`

Reply:
`__loaded` is no longer in use so I removed the styles